### DISCUSSION TITLE
Disable public access?

### DISCUSSION THREAD

**Discussion by antiquealive** [rejetto/hfs#337] [Q&A]
hi, thanks for this great software. i am completely new to networks, zero knowledge.
i just use hfs to transfer some photos from iphone to windows. 

Once launched, the bat panel offers two servers:
Serving: 10.x.x.x:port
Serving: 192.168.x.x:port

what is that 10.x.x.x address? 
i don't want any device not under my router to have access. only Lan device and the computer have hfs installed can have access. how to config? 
thanks!

**Comment by rejetto** [comment]
short: there's nothing to worry about.
long:
This is tricky stuff, I'm not sure I can make it "easy" enough, but I'll try.
"10" is another address used for local networks. There's really no difference of usage between 192 and 10 (that i know of), except that 192 is more common.
In facts, in the past I've also replaced 192 address with a 10, just because it's nicer and shorter. So, I had just 10 and no 192. Just like "mike" and "bob", names to refer to something.
Having 192 and 10 means you have another network "interface" on your computer, but this too is just local, so to say, because 10 is for local.
Your situation is common, because computers have (mostly) one physhical interface (that is, with a cable, or antenna=wifi), and 1 or a few virtual interfaces, introduced by a software you installed (or even the OS) to offer some capabilities. For example: they are used for VPN, or with Virtual Machines. In these examples, you need to communicate with these entities (vpn and vm), and instead of defining a "new way" to communicate, they simulate a network card, so they act in the same old way.

So you ask if it's "not under my router". In the example of the VPN, the VPN is communicating over the internet, but its traffic is "inside" your 192, so it's physically connected to your router, yes, but the VPN itself cannot know, it lives in its virtual world.
In the example of the VM instead, the 10 will stay just within your computer and never reach the rest of your network. Even safer, if you want.

This was not easy to write, I hope it is at least partially clear. :)
